backend/src/middleware/handleHtmlContent/hashtags/extractHashtags.js

```diff
@@ -0,0 +1,28 @@
+import cheerio from 'cheerio'
+// formats of a Hashtag:
+//   https://en.wikipedia.org/w/index.php?title=Hashtag&oldid=905141980#Style
+// here:
+//    0. Search for whole string.
+//    1. Hashtag has only 'a-z', 'A-Z', and '0-9'.
+//    2. If it starts with a digit '0-9' than 'a-z', or 'A-Z' has to follow.
+const ID_REGEX = /^\/search\/hashtag\/(([a-zA-Z]+[a-zA-Z0-9]*)|([0-9]+[a-zA-Z]+[a-zA-Z0-9]*))$/g
+
+export default function(content) {
```

Comment:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-07-01T22:38:15Z" title="Tuesday, July 2nd 2019, 12:38:15 am +02:00">Jul 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,22 @@
+import cheerio from 'cheerio'
+const ID_REGEX = /\/search\/hashtag\/([\w\-.!~*'"(),]+)/g
+
+export default function(content) {
```


Did you test `https://www.example.org/#anchor` for false positives? I am pretty sure the used method does handle that correctly - just to make sure we dont run into problems here

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-02T06:12:40Z" title="Tuesday, July 2nd 2019, 8:12:40 am +02:00">Jul 2, 2019</time>_

---


Good idea. @ulfgebhardt 
I will do tomorrow …

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-08T06:43:45Z" title="Monday, July 8th 2019, 8:43:45 am +02:00">Jul 8, 2019</time>_

---


Done @ulfgebhardt ❤️